.ci/jenkins/dsl/jobs.groovy

```diff
@@ -0,0 +1,38 @@
+import org.kie.jenkins.jobdsl.templates.KogitoJobTemplate
+
+def getDefaultJobParams(String repoName = 'optaweb-employee-rostering') {
+    return KogitoJobTemplate.getDefaultJobParams(this, repoName)
+}
+
+Map getMultijobPRConfig() {
+    return [
+        parallel: true,
+        buildchain: true,
+        jobs : [
+            [
+                id: 'optaweb-employee-rostering',
+                primary: true,
+            ]
+        ],
+    ]
+}
+
+setupMultijobPrDefaultChecks()
```

Comment:
Does the default check involve the SonarCloud analysis? See https://github.com/kiegroup/optaweb-employee-rostering/pull/677/files#diff-cc7fa003f07626ab6b3e955a24d87913c05aee8bb6c56fc05bf243f617fac962L37

Reply:
I guess we need to add that in the Jenkinsfile.buildchain. Let me try something